### ISSUE TITLE
Conflict with google/gax

### ISSUE THREAD

**Issue by ismailbaskin** [google/google-api-php-client#1253]
Please tag new version. There is conflict with google/gax package due to google/auth dependency.

**Comment by bshaffer** [comment]
Hey @ismailbaskin! Sorry for the delay, we will be having a new version coming this week, and are just waiting on a few new changes.

In the meantime, you can add this library to composer.json like this:

```json
{
    "require": {
        "google/api-client": "dev-master as v2.1.3"
    }
}
```

**Comment by bshaffer** [comment]
This is now resolved! You can update to [`v2.2.0`](https://github.com/google/google-api-php-client/releases/v2.2.0)
